docs/team/merlinlim.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Merlin Lim's Project Portfolio Page
+---
+
+## Project: Productiv
+
+Productiv is a desktop application for product managers to organise their product-related information. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Delete command for Meeting
+    * Delete command for meeting allows users to delete meeting from list.
```

Comment:
Should be "...delete **a** meeting **from displayed** meeting list." ("displayed" is optional).
Same for line 19 under edit command